packages/cms-lib/schema.js

```diff
@@ -25,11 +25,39 @@ const logSchemas = schemas => {
   logger.log(data.length ? table(data, tableConfig) : 'No Schemas were found');
 };
 
-const writeSchemaToDisk = (schema, dest) => {
+const cleanSchema = schema => {
```

Comment:
Not sure if there's a different way we wanna do this, I just took @anthmatic example from the Slack thread and grabbed those properties.  I tested and am able to create new schemas from the cleaned ones so it seems to work well

Reply:
I think we should add some unit tests for this.